ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -82,6 +82,10 @@ message ServiceDescriptor {
   //   - Key: "ni/service.tags"
   //   - Expected format: serialized JSON string of an array of strings
   //   - Example: "[\"powerup\", \"current\"]"
+  // - Programminglanguage
```

Comment:
Follow this to fix the PR checks: https://github.com/ni/measurementlink-python/blob/main/CONTRIBUTING.md#updating-grpc-stubs-when-a-proto-file-is-modified